team-D-description.md

```diff
@@ -9,3 +9,8 @@
 관심 컴포넌트 : backend.ai-manager, backend.ai.storage-proxy, backend.ai.web-server   
 간단 자기소개 : 안녕하세요. 컴퓨터공학부 4학년에 재학중인 24살 이희광입니다. 저는 벡엔드와 빅데이터와 같은 인공지능에 관심을 가지고 있습니다. 그래서 다음학기부터는 자대 석사 예약입학이 진행되어 대학원 공부도 시작될 예정이며 스타트업에서 안드로이드 개발자 업무도 병행하고 있습니다.  개발을 하면서 깃과 깃허브는 곧 잘 사용하긴했지만 제가 사용하는 기능이 매우 제한적이였기에 제대로 사용하는것같다는 느낌을 받지 못했는데 그래서 이번에 이 오픈소스 프로젝트에 참여하여 제대로 공부해보고자 하였습니다. 다행히 합격하게 되어 멘티분들과 함께 공부할수있게되고 멘토분들의 지도를 받을 수 있는것에 매우 감사하게 생각합니다.   
 
+
+
+이름 : 백종현   
+관심 컴포넌트 : backend.ai-manager, backend.ai.web-server   
```

Comment:
I found a simple typo error in line 15.
Please update the component name `backend.ai.web-server` to `backend.ai-webserver` so that everyone can know the exact name of the repo.

Reply:
I've fixed it :)